.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  IMAGE_TAG: ${{ github.sha }}
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build Docker images
+      run: |
+        set -e
+        TARGET=prod make build
+        TARGET=dev make build
+
+    - name: Lint
+      run: make lint
+      if: always()
```

Comment:
Meant? Looks like it will be forced to run even if the workflow is cancelled.

Reply:
Only alternative I see to execute in spite of other steps failing would be `success() or failure()` which seems confusing?